podcasts/ABTest/ABTestProvider.swift

```diff
@@ -0,0 +1,27 @@
+import AutomatticTracks
+
+struct ABTestProvider: ABTestProviding {
+    static let shared = ABTestProvider()
+
+    func variation(for abTest: ABTest) -> Variation {
+        ExPlat.shared.experiment(abTest.rawValue) ?? .control
```

Comment:
For `control` variants, `GET /assignments` [endpoint](https://app.swaggerhub.com/apis/yanir/experiments/0.1.0#/assignment/get_assignments__platform_) returns `null`. Glad to see this is checking for that and managing the `.control` variant.

Keep in mind that both users that assigned `control` and users that are not eligible for the experiment both are assigned `null` variations. We don't differentiate, so please do not use the `variation` value in future Tracks event  props.

It may be worth adding a comment here so others know that users assigned to `control` and users ineligible for the experiment are given the assignment `null` which corresponds to them experiencing the `control` variation.

Reply:
> We don't differentiate, so please do not use the variation value in future Tracks event props.

If I got it right it means that we should **never** send `variation` as a property in an event, right? If we want to set the property what would be the correct way of doing it?

Reply:
The way you are setting it is the correct way. Just be careful with interpreting those Tracks events, as a12s may think that this is the variation assigned vs the variation experienced.